exercise2/Statically.java

```diff
@@ -1,5 +1,32 @@
 package exercise2;
 
 public class Statically {
-    //Once you have finished getting statistically.js to run, refactor the code here in Java.
+    //Once you have finished getting statistically.js to run,
+    //refactor the code here in Java.
+    public static void main(String[] args) {
+        Statically stats = new Statically("Epoxy", "Jello");
+        stats.moveAndShake();
+        stats.stubborn();
+        stats.explain();
```

Comment:
The point of static in java is that the method can be called without instantiating an instance like you did on line 7. So you should just have the method names by themselves in main()